lib/modules/apostrophe-pieces/lib/api.js

```diff
@@ -616,6 +616,8 @@ module.exports = function(self, options) {
         // add the new field to the columns
         self.columns.push(newColumn);
       });
+      // So we do not inadvertently modify it for other subclasses
```

Comment:
Is it to be sure `listProjection` will always be an object?

Reply:
Yes there is a default setting for it.